model/src/main/java/jetbrains/jetpad/model/transform/Transformers.java

```diff
@@ -825,6 +825,76 @@ protected void doRemove() {
     };
   }
 
+  /**
+   * Select only those with the highest priority. Warning: target collection
+   * is not protected from outside writes.
+   * @param getPriority The greater is number, the higher is priority.
+   */
+  public static <ItemT>
+  Transformer<ObservableCollection<ItemT>, ObservableCollection<ItemT>> highestPriority(final Function<ItemT, Integer> getPriority) {
+    return new BaseTransformer<ObservableCollection<ItemT>, ObservableCollection<ItemT>>() {
+      @Override
+      public Transformation<ObservableCollection<ItemT>, ObservableCollection<ItemT>> transform(ObservableCollection<ItemT> from) {
+        return transform(from, new ObservableArrayList<ItemT>());
```

Comment:
Also, using list rises a question about relative order of "from" and "to" collection elements in case if "from" is a list